src/functions/module_filter.ts

```diff
@@ -66,6 +152,136 @@ export const FILTER = {
   isExported: true,
 } satisfies AddFunctionDescription;
 
+// -----------------------------------------------------------------------------
+// SORT
+// -----------------------------------------------------------------------------
+export const SORT: AddFunctionDescription = {
+  description: _t("Sorts the rows of a given array or range by the values in one or more columns."),
+  args: [
+    arg("range (range)", _t("The data to be sorted.")),
+    arg(
+      "sort_column (number, range<number>, repeating)",
+      _t(
+        "The index of the column in range or a range outside of range containing the values by which to sort."
+      )
+    ),
+    arg(
+      "is_ascending (boolean, repeating)",
+      _t(
+        "TRUE or FALSE indicating whether to sort sort_column in ascending order. FALSE sorts in descending order."
+      )
+    ),
+  ],
+  returns: ["RANGE"],
+  computeValueAndFormat: function (
+    range: Matrix<ValueAndFormat>,
+    ...values: (ValueAndFormat | Matrix<ValueAndFormat>)[]
+  ): Matrix<ValueAndFormat> {
+    assert(
+      () => values.length % 2 === 0,
+      _t(`Wrong number of Argument[]. Expected an odd number of Argument[].`) //'odd' as we should have data + pairs of (sortColumn, isAscending)
+    );
+    const _range = transposeMatrix(range);
+    return transposeMatrix(sortMatrix(_range, this.locale, ...values));
+  },
+  isExported: true,
+};
+
+// -----------------------------------------------------------------------------
+// SORTN
+// -----------------------------------------------------------------------------
+export const SORTN: AddFunctionDescription = {
+  description: _t("Returns the first n items in a data set after performing a sort."),
+  args: [
+    arg("range (range)", _t("The data to be sorted.")),
+    arg("n (number, default=1)", _t("The number of items to return.")),
+    arg(
+      "display_ties_mode (number, default=0)",
+      _t("A number representing the way to display ties.")
+    ),
+    arg(
+      "sort_column (number, range<number>, repeating)",
+      _t(
+        "The index of the column in range or a range outside of range containing the values by which to sort."
+      )
+    ),
+    arg(
+      "is_ascending (boolean, repeating)",
+      _t(
+        "TRUE or FALSE indicating whether to sort sort_column in ascending order. FALSE sorts in descending order."
+      )
+    ),
+  ],
+  returns: ["RANGE"],
+  computeValueAndFormat: function (
+    range: Matrix<ValueAndFormat>,
+    ...values: (ValueAndFormat | Matrix<ValueAndFormat>)[]
+  ): any {
+    const n = toScalar(values[0]) ?? { value: 1 };
+    assert(
+      () => typeof n.value === "number" && n.value > 0,
+      _t(`Wrong value of 'n'. Expected a strictly positive number. Got %s.`, toString(n.value))
+    );
+    const _n = toNumber(n.value, this.locale);
+    const displayTiesMode = toScalar(values[1]) ?? { value: 0 };
```

Comment:
`displayTiesMode` is not supposed to be a range (the compiler should prevent it)